### ISSUE TITLE
Rancher 3-node HA setup cannot recover after loss of 1 host

### ISSUE THREAD

**Issue by bdentino** [rancher/rancher#5539]
**Rancher Version:**
1.1.2
**Docker Version:**
1.11.2
**OS and where are the hosts located? (cloud, bare metal, etc):**
RancherOS, AWS m3.large
**Setup Details: (single node rancher vs. HA rancher, internal DB vs. external DB)**
3-node HA rancher, external db
**Environment Type: (Cattle/Kubernetes/Swarm/Mesos)**
Cattle (issue with HA environment itself)
**Steps to Reproduce:**
Cannot reliably reproduce, but in general after setting up an HA deployment, if i keep cycling hosts I will eventually get into some state from which I cannot recover
**Results:** 
This time, removing one of my hosts brought down the whole HA deployment. I am struggling to bring it back up because redis and zk simply will not start on any new hosts I add.
**Expected:**
Loss of one host should not topple a 3-node HA setup. New hosts should successfully join HA environment.

**Comment by aemneina** [comment]
Hey @bdentino can you share the docker logs for zookeeper and redis. It might reveal what's preventing them from starting.

**Comment by bdentino** [comment]
Sorry, those instances are long gone. I eventually gave up on HA because I just couldn't get it reliably to come up, stay up, and recover when testing with failed nodes. Instead set up a blue-green deployment of two single-node servers behind HAProxy for failover and it's good enough for my needs. I'm not entirely sure if the HA headache was due to instability within rancher or poor understanding of configuration options on my part. Probably some combination of both, but if no one else can reproduce this by randomly terminating HA nodes I'll close this issue.
